index.js

```diff
@@ -91,6 +92,21 @@ module.exports = function(input, inputMap) {
 			});
 			return;
 		}
+
+		if (params.includeModulePaths) {
+			// Ensure module paths include full path from current working directory
+			var pwd = process.cwd();
+			if (context.substring(0, pwd.length) === pwd) {
```

Comment:
Do you mean turn it into a named function and place it at the top of the loader function (after line 20)?